integration-tests/constants/webookRequestParameters.ts

```diff
@@ -0,0 +1,22 @@
+export interface DataRequestParams {
```

Comment:
This should live in the `types` folder, and I'd rename the file to have the same name as the main type i.e. `dataRequestParams.ts` to avoid confusion.